bundles/paikkatietoikkuna/admin/admin-layeranalytics/LayerAnalyticsDetails.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Table, Space } from 'antd';
+import { Message } from 'oskari-ui';
+import { ArrowLeftOutlined } from '@ant-design/icons';
+
+import 'antd/es/table/style/index.js';
+
+const dateLocale = 'fi-FI';
+const localeDateOptions = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit'
+};
+
+const formatTimestamp = (timestamp) => {
+    let date;
+    if (typeof timestamp !== 'undefined') {
+        date = new Date(timestamp);
+    }
+    return formatTime(date) + ' ' + formatDate(date);
+};
+const formatDate = (date) => {
+    if (typeof date === 'undefined') {
+        return '--.--.----';
+    }
+    return date.toLocaleDateString(dateLocale, localeDateOptions);
+};
+
+const formatTime = (date) => {
+    if (typeof date === 'undefined') {
+        return '--:--:--';
+    }
+    return date.toLocaleTimeString(dateLocale).replace(/\./g, ':');
+};
+
+export const LayerAnalyticsDetails = ({ layerData, closeDetailsCallback, toScaleCallback }) => {
```

Comment:
Move toScaleCallback to this file. I don't see any reason why it should be on the other file since this is the only one using it.

Reply:
Moved.